react/react1/week1/Todo/src/components/TodoList/TodoList.jsx

```diff
@@ -0,0 +1,19 @@
+import TodoListItem from "../TodoListItem/TodoListItem"
+import './TodoList.css'
+
+const TodoList = ({todo}) => {
+  const elements = todo.map(item => {
+    const {id, ...itemProps} = item;
```

Comment:
really good use of the … operator 🥇 Good to see you use the map function, which makes this component much more scalable.